pkg/perfcounter/counter.go

```diff
@@ -15,43 +15,42 @@
 package perfcounter
 
 import (
-	"sync/atomic"
-
 	"github.com/matrixorigin/matrixone/pkg/common/mpool"
+	"github.com/matrixorigin/matrixone/pkg/util/metric/stats"
 )
 
 type Counter struct {
```

Comment:
Call it CounterSet?   Reduce name confusions.

Reply:
I completely agree. @reusee please do the needful.

Reply:
@arjunsk just change it in this PR>